tasks/services/plans.js

```diff
@@ -1,6 +1,11 @@
 const feathersKnex = require('feathers-knex')
-const { iff } = require('feathers-hooks-common')
 import { isEmpty, ifElse, is, assoc, prop, map, pipe, __ } from 'ramda'
+const { iff, validateSchema } = require('feathers-hooks-common')
+const Ajv = require('ajv')
+const taskPlanSchema = require('../schemas/taskPlan')
+const taskRecipeSchema = require('../schemas/taskRecipe')
+const ajv = new Ajv({$data: true})
```

Comment:
i wonder if this should be a singleton somewhere in a module. like we export the `ajv` object with all the schemas added in `app/schema.js`. then we'd require that here.